src/saving/serializers/RegistryTypeStringConverter.cs

```diff
@@ -201,6 +201,14 @@ public class EnzymeStringConverter : RegistryTypeStringSingleTypeConverter<Enzym
     protected override string TypeName => "enzyme";
 }
 
+/// <summary>
+///   Specific converter for <see cref="Meteor"/>
+/// </summary>
+public class MeteorStringConverter : RegistryTypeStringSingleTypeConverter<Enzyme>
+{
+    protected override string TypeName => "meteor";
```

Comment:
It seems you didn't add this to `SupportedRegistryTypes` so this converted to me seems like it isn't hooked up correctly.

Reply:
This is marked as resolved but I still do not see the change that should be in the diff earlier in this file...

Reply:
I removed it completely